src/cli/rustup_mode.rs

```diff
@@ -1096,10 +1096,7 @@ fn show_active_toolchain(cfg: &Cfg<'_>, verbose: bool) -> Result<utils::ExitCode
                 )?;
             }
         }
-        None => writeln!(
-            cfg.process.stdout().lock(),
-            "There isn't an active toolchain"
-        )?,
+        None => return Err(anyhow!("there isn't an active toolchain")),
```

Comment:
Maybe tweak this wording to "there is no active toolchain" while you're in here?

Reply:
Simply "no active toolchain" would be fine, I think? That should be rendered as `error: no active toolchain` and would be consistent with the other no active toolchain error message.

Reply:
Yup, that's probably better.